### ISSUE TITLE
Checking of language tag may be too strict

### ISSUE THREAD

**Issue by rdeltour** [IDPF/epubcheck#615]
It was reported in #614 that a language code `xml:lang="eng"` triggers the message:

```
Content file has invalid language value at attribute xml:lang.
```

This seems to be OK as per [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt):

> When languages have both an ISO 639-1 two-character code and a three-
   character code (assigned by ISO 639-2, ISO 639-3, or ISO 639-5), only
   the ISO 639-1 two-character code is defined in the IANA registry

However, three-character codes are legit in some cases (e.g. `haw` for Hawaiian).

We should make sure that validation of language codes does **not** require two-language codes.

**Comment by caramelra** [comment]
Hi,

I also received this exact message while using the following 
xml:lang="he"

Please would you check it?

Thanks,

Carmel

**Comment by rdeltour** [comment]
@caramelra will do, thanks for the info!

**Comment by carmelra** [comment]
Thanks a lot!
I also noticed I'm getting the message "Content file has invalid language definition at attribute lang."
while using lang="he".
Maybe it's related to the same issue.

Carmel

**Comment by rdeltour** [comment]
I lowered the severity of the language tag check, since the rules defined in RFC  5646 are more complex than what was implemented.
I also added the `he` language code, which was not accepted previously.
